doc(release): add release note for centreon-collect-21.10.3 (#1818)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-21.10/releases/centreon-core.md

##  Centreon Collect

### 21.10.3

Release date: ``

#### Centreon Engine

##### Improvements

- Improved security of the gRPC API by listening on 127.0.0.1 by default

#### Centreon Broker

##### Improvements

- Improved security of the gRPC API by listening on 127.0.0.1 by default
- Empty strings are now accepted for parameter values in stream connectors

##### Bug fixes

> - Fixed an issue that caused the RRD rebuild mechanism to fail

Author: ```suggestion
- Fixed an issue causing pollers to be displayed as running, and resources to be displayed in Resources Status after stopping centengine
- Fixed an issue that caused the RRD rebuild mechanism to fail
```

##  Centreon Collect

### 21.10.3

> Release date: ``

Reviewer: ```suggestion
Release date: `September 27, 2022`
```

##  Centreon Collect

### 21.10.3

Release date: ``

#### Centreon Engine

##### Improvements

- Improved security of the gRPC API by listening on 127.0.0.1 by default

#### Centreon Broker

##### Improvements

- Improved security of the gRPC API by listening on 127.0.0.1 by default

> - Allow empty strings for parameter values in stream connectors

Author: ```suggestion
- Empty strings are now accepted for parameter values in stream connectors
```

##### Bug fixes

- Fixed an issue that caused the RRD rebuild mechanism to fail
- Fixed an issue that caused BAM Business activities with “Ignore the indicator in the calculation” as planned downtime calculation method to stop ignoring downtimed KPIs when they had overlapping downtimes
- Fixed a bug causing Broker to ignore additional arguments
- [Configuration] Extended the size of the URL, Notes and Action URL fields to avoid truncating long URLs

### 21.10.3

Release date: ``

### Centreon Engine

##### Improvements

> - Improve security of the gRPC API by listening on 127.0.0.1 by default

Reviewer 2: ```suggestion
- Improved security of the gRPC API by listening on 127.0.0.1 by default
```

### Centreon Broker

##### Improvements

> - Improve security of the gRPC API by listening on 127.0.0.1 by default

Reviewer 2: ```suggestion
- Improved security of the gRPC API by listening on 127.0.0.1 by default
```

> - Allow empty strings for parameter values in stream connectors

Reviewer 2: ```suggestion
- Allowed empty strings for parameter values in stream connectors
```

> ### 21.10.3
> 
> Release date: ``
> 
> #### Centreon Engine
> 
> ##### Improvements
> 
> - Improved security of the gRPC API by listening on 127.0.0.1 by default
> 
> #### Centreon Broker
> 
> ##### Improvements
> 
> - Improved security of the gRPC API by listening on 127.0.0.1 by default
> - Allow empty strings for parameter values in stream connectors
> 
> ##### Bug fixes
> 
> - Fixed an issue that caused the RRD rebuild mechanism to fail
> - Fixed an issue that caused BAM Business activities with “Ignore the indicator in the calculation” as planned downtime calculation method to stop ignoring downtimed KPIs when they had overlapping downtimes
> - Fixed a bug causing Broker to ignore additional arguments
> - [Configuration] Extended the size of the URL, Notes and Action URL fields to avoid truncating long URLs
> 
> 

Author: ```suggestion
```
